### DISCUSSION TITLE
Example of displaying context menu using the Table's onRow property

### DISCUSSION THREAD

**Discussion by AsuraKev** [ant-design/ant-design#45149] [Q&A]
Hello people,

I am wondering if Antd Table compnent can do context menu together with the Dropdown component? I see that theres a on context menu method inside the onRow props but could someone share a basic example of how to trigger the Dropdown component to show on right click? thanks!

![image](https://github.com/ant-design/ant-design/assets/52765572/c736e256-c6a6-4d96-a89c-e3936e046c05)

**Comment by AsuraKev** [comment]
got it working. if someone wants it heres a sandbox example -> https://codesandbox.io/s/virtual-list-antd-5-9-4-forked-crh4dl?file=/Dropdown.tsx

**Reply by vinikatyal** [reply]
Does this work for u?

**Reply by AsuraKev** [reply]
Yes

**Reply by vinikatyal** [reply]
Yes works i was looking if it could be be possible to click one of the action buttons and open the dropdown

**Reply by AsuraKev** [reply]
Not sure about that. But the purpose of context menu is to click inside the table to trigger
